_posts/previous/2012-06-29-video-plays-key-role-expanding-linaros-community-building-future-linux-arm.md

```diff
@@ -19,11 +19,11 @@ HONG KONG - 29 JUN 2012
+The entire event was broadcast on YouTube by the [Novacut](https://launchpad.net/novacut) video crew and video blogger Nicolas Charbonnier (Charbax) and, to make Linaro Connect more accessible in real time, all the key sessions were done using Google Hangouts. During this event a concentrated effort was made to make the event more accessible for those attending remotely by Linaro’s use of Google + Hangouts on Air () to allow for better audio and visual remote participation. This was the first time Google+ Hangouts on Air was used to record all the sessions and there were many challenges that had to be addressed, such as finding a way to let participants know where the sessions could be found ahead of time, effective room set up, camera locations and how to set up the hangouts to go to the same youtube channel all had to be resolved during the week. There were many lessons learned during this event and much feedback was received from those who participated remotely to help improve these efforts for our next event.
 
 Charbax interviewed attendees, captured sessions, demos and more from the event. Novacut is no stranger to Linaro Connect events and they have been assisting Linaro in capturing these events through video, pictures and interviews for the past year, however during this event there was an effort to video every session and activity as opposed to just key ones. Charbax joined us for the first time in Hong Kong and has already made a great impression on those who have seen his videos from the event. Most notable was the demonstration of the toolchain improvements to Linaro’s Android Evaluation Build (LEB) that was demonstrated on the PandaBoard but all the toolchain improvements are applicable to all our member boards. This video ([http://armdevices.net/2012/06/02/linaro-improvements-to-android-4-0-4-performance-on-the-pandaboard-ti-omap4430-platform/ ](http://armdevices.net/2012/06/02/linaro-improvements-to-android-4-0-4-performance-on-the-pandaboard-ti-omap4430-platform/)) has already generated more than 200,000 views.
 
-Overall between Novacut (<http://www.youtube.com/user/LinaroOrg>), Charbax[(http://armdevices.net/category/tradeshows/linaro-connect/](http://armdevices.net/category/tradeshows/linaro-connect/)) and the Google +Hangouts on Air ([]()) more videos will be added to the Linaro Connect Q2.12 Resource page by the end of July.
+Overall between Novacut (<http://www.youtube.com/user/LinaroOrg>), Charbax) and the Google +Hangouts on Air ([]()) more videos will be added to the Linaro Connect Q2.12 Resource page by the end of July.
```

Comment:
Remove the `([]())` completely?

Reply:
I'll remove it. But that was already there before I modified the file in this commit so I didn't necessarily scour the entire post.

Reply:
😁👍